gcp/appengine/frontend3/src/styles.scss

```diff
@@ -163,6 +163,40 @@ pre {
   opacity: 1;
 }
 
+.tag {
```

Comment:
Can we rename it to something more descriptive, like `severity-tag` or something?

Reply:
Thinking a bit more, these styles are used for both severity and fix tags. I reckon using more general name like `tag` is more descriptive. What do you think?

Reply:
Then shall we name it to something like `attribute-tag`? Or adding some comments for this class?